tests/http_spec.rs

```diff
@@ -311,7 +311,11 @@ impl HttpIO for MockHttpClient {
       response.body = Bytes::from_iter(body);
       Ok(response)
     } else {
-      let body = serde_json::to_vec(&mock_response.0.body)?;
+      let body = if let Value::String(x) = mock_response.0.body {
+        string_to_bytes(&x)
+      } else {
+        serde_json::to_vec(&mock_response.0.body)?
+      };
```

Comment:
Can you explain why we need this change?

Reply:
it lets us mock an endpoint that returns a GraphQL schema in `tests/http/showcase.yml`